docs/pages/desktop-access/getting-started.mdx

```diff
@@ -96,12 +99,14 @@ Copy the token to the Linux host where you will run the Desktop service as `/tmp
 
 (!docs/pages/includes/install-linux.mdx!)
 
-Create `/etc/teleport.yaml` and configure it for Desktop Access:
+Create `/etc/teleport.yaml` and configure it for Desktop Access. Update the `proxy_server`
+value to your Teleport proxy service or cloud tenant, and put the Windows machine address
+under `non_ad_hosts`:
 
 ```yml
 version: v3
 teleport:
-  nodename: example.teleport.com
+  nodename: examplewindowsaccess.teleport.com
```

Comment:
Since we don't actually own `teleport.com`, let's make sure our examples use subdomains of example.com
```suggestion
  nodename: windows.teleport.example.com
```